Kernel/Bus/PCI/Device.cpp

```diff
@@ -110,10 +111,10 @@ PhysicalAddress Device::msix_table_entry_address(u8 irq)
 
     VERIFY(index < m_interrupt_range.m_irq_count);
     VERIFY(index >= 0);
-    auto table_bar_ptr = PCI::get_BAR(device_identifier(), static_cast<PCI::HeaderType0BaseRegister>(m_pci_identifier->get_msix_table_bar())) & PCI::bar_address_mask;
+    auto table_bar_paddr = PCI::get_bar_address(device_identifier(), static_cast<PCI::HeaderType0BaseRegister>(m_pci_identifier->get_msix_table_bar())).release_value_but_fixme_should_propagate_errors();
```

Comment:
Can we get a better variable name than `paddr`?

Reply:
the abbreviations `paddr`/`vaddr` are quite common in the kernel.